packages/rich-text/src/to-dom.js

```diff
@@ -251,6 +251,15 @@ export function applyValue( future, current ) {
 	}
 }
 
+function isRangeEqual( a, b ) {
+	return (
+		a.startContainer === b.startContainer &&
```

Comment:
Can we use `isShallowEqual` instead?

Reply:
Seems like a bit more work? We only need to compare these four properties and don't need to loop over all.

Reply:
But I'm fine with this actually...

Reply:
If we're optimizing for performance, I think listing out the properties is probably going to win out, even if a bit more verbose. `isShallowEqual` is fast, but probably not _quite_ as fast.

Reply:
I'm fine with either. At the end of the day, what's one additional loop in all of the code being executed on key press. :)

Reply:
I do think it's nice to avoid adding more dependencies though.